fendermint/vm/interpreter/src/fvm/state/exec.rs

```diff
@@ -156,14 +156,17 @@ pub struct FvmUpdatableParams {
 pub type MachineBlockstore<DB> = <DefaultMachine<DB, FendermintExterns<DB>> as Machine>::Blockstore;
 
 /// A state we create for the execution of all the messages in a block.
-pub struct FvmExecState<DB>
+pub struct FvmExecState<DB, M = fendermint_module::NoOpModuleBundle>
 where
     DB: Blockstore + Clone + 'static,
+    M: ModuleBundle,
```

Comment:
Perhaps, this constraint could be relaxed to specific module traits rather than the whole bundle (also in trait implementations).